File: solutions/C/leap/1/leap.c
#include "leap.h"

bool leap_year(int year){
    if(year % 4 == 0 && year % 100 !=0) {
        if(year % 400 !=0){
            return true;
        } else return false;
    } else {return false;
    }
}

File: solutions/Python/leap/1/leap.py
def leap_year(year):
    if year % 400 == 0:
       return True 
    elif year % 100 == 0:
        return False
    elif year % 4 == 0:
        return True
    return False

   
# def leap_year(year):
   #  if (year % 4 ==0) and year % 400 == 0:
      #   print("it's a leap year")
    # elif:
     #    (year % 100 >=0)